### PR TITLE
CCE / CIS numbers for SLE 12/15 rules which belong to CIS group Initi…

### PR DESCRIPTION
**Pull request by rumch-se**
…al setup

#### Description:

- _Assignment of CCE/CIS numbers to SLE 12/15 rules which are part of CIS group 1 Initial Setup _

#### Rationale:

- Some of the SLE 12/15 rules do not have CCE numbers assigned or their CIS numbers have to be updated 

#### Review Hints:

- _Review hints here. Replace this text. Don't use the italics format!_

- _Use this optional section to give any relevant information which could help the reviewer to more quickly and assertively understand and test the changes._

- _Good examples are useful commands, if it is better to review all commits together or in a suggested sequence, any relevant discussion in other PRs or issues, etc._

### CODE REVIEW

**Review by marcusburghardt — CHANGES_REQUESTED** [review]
Many CCEs are removed from the list but not used in any rules included in this PR.

File: shared/references/cce-sle15-avail.txt [hunk 1]
```diff
@@ -1,76 +1,3 @@
-CCE-91349-1
```
[the remark this answers is not in the document]
Comment (rumch-se) [reply shared/references/cce-sle15-avail.txt:1]: PR 9874
rule - banner_etc_motd

File: shared/references/cce-sle12-avail.txt [hunk 2]
```diff
@@ -1,84 +1,3 @@
-CCE-92227-8
```
[the remark this answers is not in the document]
Comment (rumch-se) [reply shared/references/cce-sle12-avail.txt:1]: PR 9874
rule - banner_etc_motd

**Comment by rumch-se** [comment]
Hello @marcusburghardt
I have done rebase.
Have a nice day
Rumen

**Comment by marcusburghardt** [comment]
> Hello @marcusburghardt I have done rebase. Have a nice day Rumen

Hi @rumch-se , it PR is removing more CCEs from the list than using them in the rules. Could you take a look, please?

**Comment by rumch-se** [comment]
Hello @marcusburghardt 
The rebase was done
Have a nice day
Rumen

**Comment by rumch-se** [comment]
Good morning @marcusburghardt 
I have done a rebase of this PR.
Have a nice day
Rumen

**Review by marcusburghardt — APPROVED** [review]

**Pull request merged**